CMake/ClangCompilers.cmake

```diff
@@ -22,14 +22,14 @@ ENDIF(QMC_OMP)
 # Set clang specific flags (which we always want)
 ADD_DEFINITIONS( -Drestrict=__restrict__ )
 
-SET(CMAKE_C_FLAGS     "${CMAKE_C_FLAGS} -fomit-frame-pointer -fstrict-aliasing")
-SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fomit-frame-pointer -fstrict-aliasing -D__forceinline=inline")
```

Comment:
Was this causing problems in debug build?

Reply:
Yes it appears that the first -fomit -fno-omit wins.

Reply:
CMAKE_C_FLAGS is added to all builds.
For a build release CMake invokes both $CMAKE_C_FLAGS $CMAKE_C_FLAGS_RELEASE.

Reply:
Not sure what you mean the CMAKE_C(XX)_FLAGS should not contain `-f-omit-frame-pointer` if you don't want that in the flags for the debug build.  This fixes that. While leaving this optimization in the default RelWithDebugInfo and Release.